osu.Game.Rulesets.Catch/Beatmaps/CatchBeatmap.cs

```diff
@@ -24,18 +26,21 @@ public override IEnumerable<BeatmapStatistic> GetStatistics()
                     Name = @"Fruits",
                     Content = fruits.ToString(),
                     CreateIcon = () => new BeatmapStatisticIcon(BeatmapStatisticsIconType.Circles),
+                    Ratio = fruits / (float)sum,
                 },
                 new BeatmapStatistic
                 {
                     Name = @"Juice Streams",
                     Content = juiceStreams.ToString(),
                     CreateIcon = () => new BeatmapStatisticIcon(BeatmapStatisticsIconType.Sliders),
+                    Ratio = juiceStreams / (float)sum,
                 },
                 new BeatmapStatistic
                 {
                     Name = @"Banana Showers",
                     Content = bananaShowers.ToString(),
                     CreateIcon = () => new BeatmapStatisticIcon(BeatmapStatisticsIconType.Spinners),
+                    Ratio = Math.Min(bananaShowers / 10f, 1),
```

Comment:
I guess I understand why you've done this but this baffles me on so many levels. why is the property called a "ratio" and xmldoc'd as something that seems anywhere close to useful anywhere if it's just going to be basically a vibes property for spinner-like objects capped at 10 (again, for reasons that I can understand). why not null? why are the fruits / juice stream ones ever *accounting for* banana showers (by way of dividing over the sum which includes banana showers) if the banana showers one *is not going to actually be a ratio at all*?

I dunno about any of this. I feel like I said it many times but the bars just make zero sense and seem to be just a "looks cool" thing rather than something anyone actually needs anywhere. can we just get rid?

Reply:
It really is just a "look cool" factor.

If this is a blocker, we could make it null for spinners.

Reply:
Sounds much better. I'm also not happy with `Ratio` (luckily you didn't have to see what this looked like before i screamed about the original version 😅).